src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorState.kt

```diff
@@ -1,6 +1,11 @@
 package zd.zero.waifu.motivator.plugin.settings
 
+import java.util.concurrent.TimeUnit
+
 class WaifuMotivatorState {
+    companion object {
+        val DEFAULT_IDLE_TIMEOUT = TimeUnit.MILLISECONDS.convert(5L, TimeUnit.MINUTES)
```

Comment:
question: Can we make the minimum unit to seconds instead? Even in the interface as I think that users won't input 100ms for the idle time and it would be normally set as minutes.

Reply:
That's fair. I converted it to minutes instead of seconds.